[DOCS]: added yelp clone ember-twiddle example

toranb opened the pull request:
I've been working on a yelp clone recently and decided it was a solid non trivial example that we could put up for others to checkout. Please review the twiddle as part of this pull request - I plan to tweet this out on Monday morning assuming it's not complete garbage :)

cc @cafreeman @dustinfarris @williamsbdev @brettburley

dustinfarris commented:
lol "welp" 😂 

The example looks and works great.

Is this a pod structure app?  That throws me off wherever its used, which is completely subjective criticism.  But I wonder what percentage of viewers might feel the same way.  I am reminded of previous discussions regarding ease of adoption.

I like the lodash examples in utilities/object.js

I think some kind of welcome message would be helpful.  Maybe a collapsible header area that just instructs the user what to do to get the most out of the example.

Minor stuff:

- The name of the restaurants should probably be a link
- Clicking the picture of a restaurant (which I assumed would take me to the detail page) seems to have confused Twiddle
- The review input box is pre-populated with the text from a prior review, but it feels broken, as a first-timer I didn't know if that was supposed to be the fictional me's previous review (submit button should probably read "Submit Edit" or something if this is the case)
- Tasty Tacos sometimes doesn't have a review box, esp. after initial load and clicking "5 reviews"
- The cursor for the stars widget is an I-beam which doesn't feel like I shouldn't click on it

toranb commented:
@dustinfarris great feedback - more discussion

1) about the banner - I do like this idea (as it would help guide the person using the twiddle). Any examples of css/html for the visual approach you had in mind? Sorry I'm the last guy you'd want doing design and I'll need something to break it down for my scss team

2) good catch on the photos- it's actually a dead anchor tag and I've fixed this :)

3) I like this mention of "edit" vs "post" - I did want a few user experiences to show off redux + ember and this was intended to be an existing comment/rating as you found out. I'll edit this text to clear that up a little

4) Tasty tacos was used to represent a UX in which you had to rate the item first/then (once rated) a comment textarea is loaded dynamically. I could add some text like "click to rate this restaurant" or something ? thoughts? I wanted to show an unrated restaurant here (that was the goal anyway)

5) good catch - I've added the pointer class so you now get a hand when you hover :)

side note: I did choose to use pods and in doing so I realize it's not the default for twiddle, some of the ember /core don't like it but it's what I use for all my projects. Subjective as you mention/ but I actually prefer it for organization (until the grand module unification lands anyway)

dustinfarris commented:
I was picturing something like bootstrap alerts for the welcome message http://getbootstrap.com/javascript/#alerts <http://getbootstrap.com/javascript/#alerts>

Yes, some kind of guidance for Tasty Tacos would be helpful I think.

> On Oct 31, 2016, at 9:17 PM, Toran Billups <notifications@github.com> wrote:
> 
> @dustinfarris <https://github.com/dustinfarris> great feedback - more discussion
> 
> 1) about the banner - I do like this idea (as it would help guide the person using the twiddle). Any examples of css/html for the visual approach you had in mind? Sorry I'm the last guy you'd want doing design and I'll need something to break it down for my scss team
> 
> 2) good catch on the photos- it's actually a dead anchor tag and I've fixed this :)
> 
> 3) I like this mention of "edit" vs "post" - I did want a few user experiences to show off redux + ember and this was intended to be an existing comment/rating as you found out. I'll edit this text to clear that up a little
> 
> 4) Tasty tacos was used to represent a UX in which you had to rate the item first/then (once rated) a comment textarea is loaded dynamically. I could add some text like "click to rate this restaurant" or something ? thoughts? I wanted to show an unrated restaurant here (that was the goal anyway)
> 
> 5) good catch - I've added the pointer class so you now get a hand when you hover :)
> 
> side note: I did choose to use pods and in doing so I realize it's not the default for twiddle, some of the ember /core don't like it but it's what I use for all my projects. Subjective as you mention/ but I actually prefer it for organization (until the grand module unification lands anyway)
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/toranb/ember-redux/pull/33#issuecomment-257467013>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABCWvaRROE5fmzlE4cT00FWQ1fYv6rIqks5q5pNCgaJpZM4Kj_35>.
> 

-- 
Dustin Farris

toranb commented:
@dustinfarris I'll work up a banner with a few of the UX workflows for people in the coming week -for now I'm planning to merge this so our README has both a simple/and real example app. The good/bad? part of twiddle is that you can update it and the actual url remains the same (so I can change that twiddle over time/improve it)
